tests/test.cpp

```diff
@@ -1276,6 +1276,67 @@ void ParseUnionTest() {
                         "{ e_type: N_A, e: {} }"), true);
 }
 
+void UnionVectorTest() {
+  // load FlatBuffer fbs schema.
+  std::string schemafile;
+  TEST_EQ(flatbuffers::LoadFile(
+    "tests/union_vector/union_vector.fbs", false, &schemafile), true);
+
+  // parse schema.
```

Comment:
note that you're parsing this schema, but it is not used in the code below. for this to be useful, you should also load a json file with such a vector in it?